guides/common/assembly_configuring-capsule-custom-server-certificate.adoc

```diff
@@ -11,8 +11,12 @@ To configure your {SmartProxyServer} with a custom certificate, complete the fol
 . xref:deploying-a-custom-ssl-certificate-to-{smart-proxy-context}-server_{smart-proxy-context}[]
 . xref:deploying-a-custom-ssl-certificate-to-hosts_{smart-proxy-context}[]
 
+:parent-ProductName: {SmartProxyServer}
+:ssl-common-name: {smartproxy-example-com}
 //Creating a Custom SSL Certificate for {SmartProxyServer}
 include::modules/proc_creating-a-custom-ssl-certificate.adoc[leveloffset=+1]
+:!ssl-common-name:
+:!parent-ProductName:
```

Comment:
```suggestion
:ProductName: {parent-ProductName}
:!parent-ProductName:
```

Reply:
Getting `unidentified attribute productname` in `deploy / build-html`. Other build seems to be happening fine, works locally for me too.

Reply:
Local build fails for `make browser BUILD=katello` with the same error for `productname`, but if I follow this up with a ` make browser BUILD=satellite`, that succeeds. Now if I try to build for Katello again, it succeeds.